### PR TITLE
Dropping the unused parameters

### PR DESCRIPTION
**Pull request by fbm3307**
This is to Drop unused parameters from the embedded MUR.Status.UserAccounts.Clusters type
Related PRs
API - https://github.com/codeready-toolchain/api/pull/387
Toolchain e2e - https://github.com/codeready-toolchain/toolchain-e2e/pull/855

### CODE REVIEW

File: config/crd/bases/toolchain.dev.openshift.com_toolchainconfigs.yaml [hunk 1]
```diff
@@ -619,18 +619,6 @@ spec:
...
-                            properties:
-                              ref:
-                                description: Reference is the name of the secret resource
-                                  to look up
-                                type: string
-                              virtualMachineAccessKey:
-                                description: The key in the secret values map that
-                                  contains a comma-separated list of SSH keys
-                                type: string
-                            type: object
```
[the remark this answers is not in the document]
Comment (fbm3307) [reply config/crd/bases/toolchain.dev.openshift.com_toolchainconfigs.yaml:633]: yea thought so ...will update it

[the remark this answers is not in the document]
Comment (fbm3307) [reply config/crd/bases/toolchain.dev.openshift.com_toolchainconfigs.yaml]: updated it

**Comment by MatousJobanek** [comment]
@fbm3307 could you please update also the api dependency to the modified version in your branch/fork, and remove the respective code also from the MasterUserRecord controller code?

**Comment by fbm3307** [comment]
Updated it

**Review by alexeykazakov — APPROVED** [review]

**Review by MatousJobanek — APPROVED** [review]
nice :+1: :1st_place_medal: 
Thanks a lot for cleaning it up

**Comment by MatousJobanek** [comment]
@fbm3307 I've noticed that you removed the replace clause from the go.mod file. please, don't forget to update the api dependency when you merge the api PR.

**Pull request merged**